### PR TITLE
AutoFixture.NUnit3

### PR DESCRIPTION
**Pull request by hackle**
Following up on discussions on issue #524 - this is an empty solution to start with.

### CODE REVIEW

**Comment by hackle** [comment]
No hard feelings @ploeh and thanks for sharing the term bikeshedding :)

File: Src/AutoFixture.NUnit3.sln [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 2012
```
Comment (moodmosaic) [inline Src/AutoFixture.NUnit3.sln]: This solution file targets Visual Studio 2012. However, I've opened it with Visual Studio 2015 and everything went well.

**Comment by moodmosaic** [comment]
I've pulled the branch locally and the newly added solution opens in Visual Studio without any issues.

File: Src/AutoFixture.NUnit3/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("AutoFixture.NUnit3")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Ploeh")]
+[assembly: AssemblyProduct("AutoFixture.NUnit3")]
+[assembly: AssemblyCopyright("Copyright © Ploeh 2016")]
```
Comment (ploeh) [inline Src/AutoFixture.NUnit3/Properties/AssemblyInfo.cs]: Feel free to change this to something like "Copyright © AutoFixture contributors 2016" (or similar) in a future pull request, if you'd like :smile:

Comment (ploeh) [inline Src/AutoFixture.NUnit3/Properties/AssemblyInfo.cs]: Actually, legally, I don't even know if it makes sense to have a copyright notice here, given that AutoFixture is released under the MIT licence, but IANAL...

**Comment by ploeh** [comment]
Thank you for your contribution :+1: 

One of the next steps that ought to be taken is that the _Verify_ and _Release_ build configurations should treat warnings as errors.

**Pull request merged**